.github/workflows/Build.yaml

```diff
@@ -13,7 +13,7 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-latest
-    timeout-minutes: 60
+    timeout-minutes: 90
```

Comment:
Why do we need to increase the timeout of this job if we also removed the `testDemoDebug testProdDebug` tasks?

Reply:
ack